crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,14 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def stop_chef_service
+    ssh_cmd("/usr/sbin/rcchef-client stop")
```

Comment:
We are already including the chef client, and i dont want to execute the lwrp directly, i just want to fetch the correct command from the platform mapping :)